### PR TITLE
Add Google Tag events

### PR DESCRIPTION
**Pull request by fabryscript**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+GTAG_ID=<Google tag id>
```
Comment (DavideSegullo) [inline .env.example:1]: Can we please add table inside the `README.MD` where we describe all the env file, the config ecc that we have inside the repo? Something like this:
```
| Name    | Description | 
| ——————— | ——————————— |
| GTAG_ID | …..         | 
```

It's helpful for maintenance

File: components/sections/hero/index.tsx [hunk 2]
```diff
@@ -1,4 +1,6 @@
+import { StartTradingButton } from "@/components/sections/hero/start-trading-button";
 import { cn } from "@/lib/utils";
+import { sendGTMEvent } from "@next/third-parties/google";
```
Comment (DavideSegullo) [inline components/sections/hero/index.tsx:3]: Why do we need this?

**Review by DavideSegullo — CHANGES_REQUESTED** [review]
Can we create a general wrapper for all the event links? Using `asChild` pattern like radix and shadcn does?

```tsx
<GTagLink asChild eventName="cardClicked" label="Title">
  <button ...>...</button>
</GTagLink>
```

File: components/sections/hero/index.tsx (hunk 2 above)
Comment (fabryscript) [reply components/sections/hero/index.tsx]: I forgot to remove it :)

File: .env.example (hunk 1 above)
Comment (fabryscript) [reply .env.example:1]: [c1fd3c4](https://github.com/osmosis-labs/osmosis-website/pull/60/commits/c1fd3c4eefe74e952a116ad835181eb7317337f2)

**Comment by fabryscript** [comment]
@DavideSegullo https://github.com/osmosis-labs/osmosis-website/pull/60/commits/45a98886983b16c2016060fa4160d723d48f5573

**Review by DavideSegullo — APPROVED** [review]

**Pull request merged**